tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -150,6 +152,133 @@ def get_pause_after_test(test_loader, **kwargs):
+                                       num_test_groups=len(
+                                           test_groups),
+                                       **run_test_kwargs)
+
+        executor_cls = product.executor_classes.get(test_type)
+        executor_kwargs = product.get_executor_kwargs(logger,
+                                                      test_type,
+                                                      test_environment,
+                                                      run_info,
+                                                      **run_test_kwargs)
+
+        if executor_cls is None:
+            logger.error("Unsupported test type %s for product %s" %
+                         (test_type, product.name))
+            continue
+
+        for test in test_loader.disabled_tests[test_type]:
+            logger.test_start(test.id)
+            logger.test_end(test.id, status="SKIP")
+            counts["skipped"] += 1
+
+        if test_type == "testharness":
+            run_tests = {"testharness": []}
+            for test in test_loader.tests["testharness"]:
+                if ((test.testdriver and not executor_cls.supports_testdriver) or
+                        (test.jsshell and not executor_cls.supports_jsshell)):
+                    logger.test_start(test.id)
+                    logger.test_end(test.id, status="SKIP")
+                    counts["skipped"] += 1
+                else:
+                    run_tests["testharness"].append(test)
+        else:
+            run_tests = test_loader.tests
+
+        recording.pause()
+        with ManagerGroup("web-platform-tests",
+                          run_test_kwargs["processes"],
+                          test_source_cls,
+                          test_source_kwargs,
+                          browser_cls,
+                          browser_kwargs,
+                          executor_cls,
+                          executor_kwargs,
+                          run_test_kwargs["rerun"],
+                          run_test_kwargs["pause_after_test"],
+                          run_test_kwargs["pause_on_unexpected"],
+                          run_test_kwargs["restart_on_unexpected"],
+                          run_test_kwargs["debug_info"],
+                          not run_test_kwargs["no_capture_stdio"],
+                          recording=recording) as manager_group:
+            try:
+                manager_group.run(test_type, run_tests)
+            except KeyboardInterrupt:
+                logger.critical("Main thread got signal")
+                manager_group.stop()
+                raise
+            counts["total_tests"] += manager_group.test_count()
+            counts["unexpected"] += manager_group.unexpected_count()
+            counts["unexpected_pass"] += manager_group.unexpected_pass_count()
+
+    return True
+
+
+def evaluate_runs(counts, iteration_timeout, run_test_kwargs):
+    """Evaluates the test counts after the
```

Comment:
No need to split this across multiple lines.

Reply:
switched to 1-liner 👍